README.md

```diff
@@ -52,13 +52,23 @@ Start the container
       --ipc=host \
       --user=root \
       --privileged \
-      -v /:/host:Z \
-      -it kubevirt/libvirt:latest
+      --volume=/:/host:Z \
+      --volume=/dev:/host-dev \
+      --volume=/var/run/libvirt:/var/run/libvirt \
+      --volume=/sys:/host-sys \
+      --tty=trye \
```

Comment:
Typo: "trye" instead of "true".

That said, libvirtd works fine when launched without access to a terminal and you're detaching the container anyway, so instead of fixing the typo just drop the line altogether.